lib/packages/docutils/rst.nim

```diff
@@ -229,6 +229,7 @@ type
     meFootnoteMismatch = "mismatch in number of footnotes and their refs: $1",
     mwRedefinitionOfLabel = "redefinition of label '$1'",
```

Comment:
add a comment: keep in snc with compiler/lineinfos.TMsgKind, and maybe re-order to make the common enums follow 1:1

Reply:
done.